_docs/concepts/security/mutual-tls.md

```diff
@@ -76,7 +76,7 @@ Service-to-service communication is tunneled through the client side [Envoy](htt
 
 Istio 0.2 supports services running on both Kubernetes pods and VM/bare-metal machines. We use different key provisioning mechanisms for each scenario.
 
-For services running on Kubernetes pods, the per-cluster Istio CA (Certificate Authority) automates the key & certificate management process. It mainly performs four critical operations :
+For services running on Kubernetes pods, the per-cluster Istio Citadel (acting as Certificate Authority) automates the key & certificate management process. It mainly performs four critical operations :
```

Comment:
Since Citadel is a proper name, please don't use Istio Citadel and just use "Citadel", here and elsewhere.

We say "Mixer does X, Pilot does Y, and Citadel does Z".

Reply:
Good catch. I will update them.